File: 1er Trimestre/EjerciciosAlgoritmo2/Ejercicio3.py
"""Algoritmo que nos permita introducir dos números por teclado (base y potencia)
y posteriormente nos calcule el valor de elevar la base a la potencia introducida
(basePotencia), el resultado se mostrará por pantalla.
La potencia la obtendremos mediante multiplicaciones sucesivas,
es decir: 7(5) = 7 * 7 * 7 * 7 * 7"""

base = int(input("Inserta un número base: "))
potencia = int(input("Inserta su potencia: "))
resultado = 1

while potencia != 0:
    potencia -= 1
    resultado = resultado * base
print(resultado)

File: ProgramasVarios/src/ejercicios002/Ejercicio3.java
package ejercicios002;
import javax.swing.*;
public class Ejercicio3 {
 public static void main(String[] args) {
	 /* Algoritmo que nos permita introducir dos números por teclado (base y potencia)
	  * y posteriormente nos calcule el valor de elevar la base a la potencia introducida
	  * (basePotencia), el resultado se mostrará por pantalla.
	  * La potencia la obtendremos mediante multiplicaciones sucesivas, es decir: 75 = 7 * 7 * 7 * 7 * 
	  */
	 int base, potencia,resultado;
	 base = Integer.parseInt(JOptionPane.showInputDialog("Introduce el valor de la base: "));
	 potencia = Integer.parseInt(JOptionPane.showInputDialog("Introduce el valor de la potencia"));
	 resultado = 1;
	 if (potencia == 0) {
	 }else {
	 	do {
	 		potencia = potencia - 1;
	 		resultado = resultado * base;
		 
	 	}while(potencia != 0 );
	 }
	 JOptionPane.showMessageDialog(null,"Vamos a ver el resultado es: " + resultado);
 }
	 }
 
 
